.travis.yml

```diff
@@ -33,29 +33,23 @@ before_script:
   - chmod a+rwx ${HOST_COVERAGE_DATA_DIR}
 
 script:
-  - docker-compose -f docker-compose.yml -f docker-compose.ci.yml run cvat_ci /bin/bash -c 'coverage run -a manage.py test cvat/apps utils/cli && mv .coverage ${CONTAINER_COVERAGE_DATA_DIR}'
```

Comment:
Can we leave it as is?

Reply:
In this case, the coverage measurement readings will alternate depending on the type of CI being triggered. This command also sends data to the coveralls server.